bevy_lint/src/lib.rs

```diff
@@ -1,11 +1,20 @@
 // Enables linking to `rustc` crates.
 #![feature(rustc_private)]
+// Allows chaining `if let` multiple times using `&&`.
+#![feature(let_chains)]
```

Comment:
This breaks rustfmt on lines using it, FYI. Not saying you should remove it, just pointing out.

Reply:
Yeah, `let_chains` are pretty much a requirement when writing lints due to how much pattern matching is required. I'll keep this in mind, though, in case `rustfmt` does anything finnicky in the future.

Reply:
It just doesn't run at all on lines using let chains, so you'll have to format those by hand.